node/builder.go

```diff
@@ -643,6 +643,10 @@ func Repo(r repo.Repo) Option {
 			Override(new(dtypes.UniversalBlockstore), modules.UniversalBlockstore),
 
 			If(cfg.EnableSplitstore,
+				If(cfg.Splitstore.ColdStoreType == "universal",
+					Override(new(dtypes.ColdBlockstore), From(new(dtypes.UniversalBlockstore)))),
```

Comment:
I find this naming confusing here. The term `universal` was used to represent that the store contains both categories of data (chain or state data), not the technology choice. We might need to revisit how these things are wired at the "LockedRepo" level. Maybe we're doing too much at DI and we could offload the choice to the repo itself?

Reply:
you named those things :)

Reply:
Yeah, `UniversalBlockstore` makes sense as the blockstore that the system uses as a whole, but not as a backing implementation for one of the splitstore stores.

Reply:
Anyway, a lot of things are messy in Lotus' DI, so I don't think getting this better structured is a blocker for now.

Reply:
punts aside, I don't think strongly about this. I am fine with doing it this way for now, and we can revisit later.